### ISSUE TITLE
Nested accordions error

### ISSUE THREAD

**Issue by GorisWW** [vladotesanovic/ngSemantic#66]
Hi.

I've encountered a problem with sm-accordion. 
If I put an accordion inside content of another one, onClick event became attached twice on child accordion's title, and its content closes immediately after opening.

Example:
```
<sm-accordion>
    <sm-accordion-item>
        <accordion-title>What is a dog?</accordion-title>
        <accordion-content>
            A dog is a type of domesticated animal. 

            <sm-accordion>
                <sm-accordion-item>
                    <accordion-title>What is a dog?</accordion-title>
                    <accordion-content>
                        A dog is a type of domesticated animal.
                    </accordion-content>
                </sm-accordion-item>
            </sm-accordion>

        </accordion-content>
    </sm-accordion-item>
</sm-accordion>
```

As a local workaround, I've overridden your classes, forcing them not to call accordion(options) and use "accordion" class instead of "ui accordion" if parent accordion is detected somewhere (results look more like an nested accordion example at SUI site).
```
if (!jQuery(this.element.nativeElement).parents("accordion-content").length)
            jQuery(this.element.nativeElement).accordion(this.smDirAccordion || {});

```

It works with a small issue, that parent accordion will override "styled" option for children.

Is there any other options?

P.S.
Thank you for your work.

**Comment by vladotesanovic** [comment]
This is fixed and published in 1.1.6 verion

You can see demo here ( nested accordion )

https://ng-semantic.herokuapp.com/#/elements/accordion

**Comment by GorisWW** [comment]
Thanks!
